_includes/v22.2/backups/destination-privileges.md

```diff
@@ -0,0 +1,20 @@
+The user that runs `BACKUP` to external storage does **not** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
+
+{% include_cached new-in.html version="v22.2" %} You can grant a user the `EXTERNALIOIMPLICITACCESS` [system privilege](security-reference/authorization.html#supported-privileges). This privilege allows a user without the `admin` role to interact with a cloud storage resource using `IMPLICIT` authentication.
+
+The user that runs `BACKUP` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```

Comment:
Maybe we can rephrase or restructure this sentence because there is now no ExternalStorage to which *only* an admin can backup, import, restore etc. If the user has `EXTERNALIOIMPLICITACCESS` even the custom endpoint and nodelocal are fair game for the non-admin user. 

We should probably structure it as:

`EXTERNALIOIMPLICITACCESS` is required for the following scenarios:

No special privilege is required in the following scenarios:

Reply:
@adityamaru OK, let me check that I understand your proposed structure... Like the following?:

EXTERNALIOIMPLICITACCESS is required for the following scenarios:

- A user without the `admin` role to interact with a cloud storage resource using `IMPLICIT` authentication.

No special privilege is required in the following scenarios:

- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
- [Userfile](use-userfile-for-bulk-operations.html)
- Use of a [custom endpoint](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) on S3.
- [Nodelocal](cockroach-nodelocal-upload.html)

Reply:
EXTERNALIOIMPLICITACCESS or admin is required for the following scenarios:

- A user without the admin role to interact with a cloud storage resource using IMPLICIT authentication.
- Use of a [custom endpoint](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) on S3.
- Nodelocal
- HTTP

No special privilege is required for:
- Amazon S3 and Google Cloud Storage using SPECIFIED credentials. 
- Azure Storage is always SPECIFIED by default.
- [Userfile](https://github.com/cockroachdb/docs/pull/use-userfile-for-bulk-operations.html)
- External Connections (external://) since they have their own privileges.

Reply:
Thanks, will make the changes.

Reply:
I will add in external connections with its PR. As this PR will merge first, there will be no documentation on that until the ext. connections work is complete.